Kernel/Storage/StorageDevice.cpp

```diff
@@ -38,14 +38,14 @@ StorageDevice::StorageDevice(Badge<RamdiskDevice>, LUNAddress logical_unit_numbe
 
 ErrorOr<void> StorageDevice::after_inserting()
 {
-    after_inserting_add_to_device_management();
     auto sysfs_storage_device_directory = StorageDeviceSysFSDirectory::create(SysFSStorageDirectory::the(), *this);
     m_sysfs_device_directory = sysfs_storage_device_directory;
     SysFSStorageDirectory::the().plug({}, *sysfs_storage_device_directory);
     VERIFY(!m_symlink_sysfs_component);
     auto sys_fs_component = TRY(SysFSSymbolicLinkDeviceComponent::try_create(SysFSBlockDevicesDirectory::the(), *this, *m_sysfs_device_directory));
     m_symlink_sysfs_component = sys_fs_component;
     after_inserting_add_symlink_to_device_identifier_directory();
+    after_inserting_add_to_device_management();
```

Comment:
slightly out of scope:
— the name of the method called above is telling exactly what is added; is it possible to update this name to also say what exactly is added to the device management? (an entry, iʼd guess?)

Reply:
I rather not as this is quite verbose already.

Reply:
i do not like sentences as the method names either but you canʼt really tell from here what will be the effect of the call without visiting the method itself (and itʼd be good not to do that if the _details_ are not needed).

but thatʼs just a hint being a hint. 🙂